specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2022-01-01/redisenterprise.json

```diff
@@ -1189,6 +1189,48 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Cache/redisEnterprise/locations/{location}/skus": {
```

Comment:
I think we shouldn't need or want the redisEnterprise path segment - it should be just "/subscriptions/{subscriptionId}/providers/Microsoft.Cache/locations/{location}/skus"